cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalApplicationTemplateManager.java

```diff
@@ -97,6 +98,7 @@ public LocalApplicationTemplateManager(CConfiguration configuration, PipelineFac
     pipeline.addLast(new ApplicationTemplateVerificationStage(store, datasetFramework, adapterService));
     pipeline.addLast(new DeployDatasetModulesStage(configuration, namespace,
                                                    datasetFramework, inMemoryDatasetFramework));
+    pipeline.addLast(new CreateDatasetInstancesStage(configuration, datasetFramework, namespace));
```

Comment:
should we add back stream creation as well?  dont have a use case right now, but if you can create datasets it seems weird that you cant create streams